package.json

```diff
@@ -18,6 +18,8 @@
     "rules-status": "eslint-index lib/config/all.js  --format table",
     "rules-omitted": "eslint-index lib/config/all.js  --status omitted",
     "lint": "eslint . --max-warnings 0",
+    "format": "prettier --write --single-quote --no-bracket-spacing --trailing-comma es5 './**/*.js'",
+    "eslint-check": "eslint --print-config .eslintrc.js | eslint-config-prettier-check",
```

Comment:
As we update `prettier` and/or `eslint` hopefully, this can continue to list conflicting rules.